packages/eslint-plugin/docs/rules/no-base-to-string.mdx

```diff
@@ -67,8 +69,8 @@ const literalWithToString = {
 ### `ignoredTypeNames`
 
 Stringified regular expressions of type names to ignore.
-This is useful for types missing `toString()` (but actually has `toString()`).
-There are some types missing `toString()` in old version TypeScript, like `RegExp`, `URL`, `URLSearchParams` etc.
+This is useful for types missing `toString()` or `toLocaleString()` (but actually has `toString()` or `toLocaleString()`).
+There are some types missing `toString()` or `toLocaleString()` in old version TypeScript, like `RegExp`, `URL`, `URLSearchParams` etc.
```

Comment:
This is an existing typo, but since you're in the area, do you mind? 😄 

```suggestion
There are some types missing `toString()` or `toLocaleString()` in old versions of TypeScript, like `RegExp`, `URL`, `URLSearchParams` etc.
```

Reply:
Of course! I fixed it in ebc6947.